### ISSUE TITLE
issue-AI: worklistAlgorithm is not using the initilalized value for bbIn["0"]

### ISSUE THREAD

**Issue by mayank393** [CS639A-PAVT/BugTracker#25]
**Describe the bug**
As per my understanding of the abstractInterpretations.worklistAlgorithm calls ForwardAnalysis.meet() on the predecessor blocks to form the input of any block. However, since the first block does not have any predecessor. The values I have assigned in the ForwardAnalysis.initialize() are not showing in the final formed bbIn and bbOut variables.

If I add some lines in the abstractInterpretations.worklistAlgorithm then it works correctly
```
  if label == 'T':
      inlist.append(bbOut[pred.name][0])
  else:
      assert len(bbOut[pred.name]) > 1
      inlist.append(bbOut[pred.name][1])

  '''Additional Code'''
  if len(predList) == 0:
      inlist.append(bbIn["0"])
```

Kindly take a look once.
Apologies for not providing a better description.

**Comment by pkalita595** [comment]
Thanks for pointing out this issue, will release a fix soon.

**Comment by pkalita595** [comment]
Fixed in Kachua v5.2
